test_user_management.py

```
@@ -0,0 +1,391 @@
+                email TEXT,
+                role TEXT DEFAULT 'user',
+                is_active BOOLEAN DEFAULT 1,
+                created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+            )
+        """)
+        
+        # Insert admin user with hardcoded credentials
+        cursor.execute(f"""
+            INSERT OR IGNORE INTO users (username, password, email, role)
+            VALUES ('{ADMIN_USERNAME}', '{ADMIN_PASSWORD}', 'admin@company.com', 'admin')
+        """)
```

Ruff (0.15.6)

[error] Possible SQL injection vector through string-based query construction

(S608)